config/default.yml

```diff
@@ -2778,6 +2778,13 @@ Style/GuardClause:
   # needs to have to trigger this cop
   MinBodyLength: 1
 
+Style/HashEachMethods:
+  Description: 'Use Hash#each_key and Hash#each_value.'
+  StyleGuide: '#hash-each'
+  Enabled: false
```

Comment:
Can you update to `Enabled: pending` status?
https://github.com/rubocop-hq/rubocop/pull/7677#discussion_r371999553